packages/babel-helpers/package.json

```diff
@@ -17,5 +17,8 @@
   },
   "devDependencies": {
     "@babel/helper-plugin-test-runner": "^7.0.0"
+  },
+  "peerDependencies": {
+    "@babel/core": "^7.0.0-0"
```

Comment:
Adding (or updating) a peer dependency is a technically a breaking change. (https://github.com/semver/semver/issues/502)

Maybe we could pass `File` as a parameter to `loadHelper` (from `get` and `ensure`), and only instantiate it if it is present?

Reply:
Good to know about that! Though I am not sure if, practically, there is anyone using `@babel/helpers` 7 with `babel-core` 6.

I end up revising `helpers.ensure` to accept extra `fileClass` because `get` interface is complicated enough.